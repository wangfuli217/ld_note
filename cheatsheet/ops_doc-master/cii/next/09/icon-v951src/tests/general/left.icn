#SRC: JCON

procedure main()

   write(left("abc"))
   write(left("def", ))
   write(left("ghi", &null))
   write(left("jkl", 2))
   write(left("mno", 3))
   write(left(237, 4))
   write(left("stu", 5))
   write(left("vwx", 6))

   write(left("abc", 3, ))
   write(left("def", , "."))
   write(left("ghi", &null, "."))
   write(left("jkl", 2, "."))
   write(left("mno", 3, "."))
   write(left(237, 4, "."))
   write(left("stu", 5, "."))
   write(left("vwx", 6, "."))

   write(left("abc", 3, ))
   write(left("def", , "<>"))
   write(left("ghi", &null, "<>"))
   write(left("jkl", 2, "<>"))
   write(left("mno", 3, "<>"))
   write(left(237, 4, "<>"))
   write(left("stu", 5, "<>"))
   write(left("vwx", 6, "<>"))
end
