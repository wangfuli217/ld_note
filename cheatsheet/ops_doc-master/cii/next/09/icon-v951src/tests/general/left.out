a
d
g
jk
mno
237 
stu  
vwx   
abc
d
g
jk
mno
237.
stu..
vwx...
abc
d
g
jk
mno
237>
stu<>
vwx><>
